docs/Install.md

```diff
@@ -7,84 +7,79 @@
 If you want to customize the installation of the Splunk Operator, download a copy of the installation YAML locally, and open it in your favorite editor.
 
 ```
-wget -O splunk-operator.yaml https://github.com/splunk/splunk-operator/releases/download/1.0.5/splunk-operator-install.yaml
+wget -O splunk-operator.yaml https://github.com/splunk/splunk-operator/releases/download/1.0.5/splunk-operator-cluster.yaml
 ```
 
+## Default Installation
 
-## Installation using a non-admin user
-
-Kubernetes only allows administrators to install new `CustomResourceDefinition` objects. All other objects included in the `splunk-operator.yaml` file can be installed by regular users within their own namespaces. 
-
-If you are not an administrator, you can have an administrator create the required objects for you by running:
-
-```
-kubectl apply -f https://github.com/splunk/splunk-operator/releases/download/1.0.5/splunk-operator-crds.yaml
-```
-
-Afterwards, you can download and use the yaml file to install the operator within your own namespace:
+By default operator will be installed in `splunk-operator` and will watch all the namespaces of your cluster for splunk enterprise custom resources:
```

Comment:
Nit: in `splunk-operator` _namespace_

Reply:
fixed

Reply:
I wasn't clear I guess. I meant 
By default operator will be installed in `splunk-operator` **namespace** and will ...

Reply:
I did that change, not sure how i missed it